pkg/controllers/namespace/namespace_controller.go

```diff
@@ -34,9 +34,10 @@ var (
 
 // NamespaceReconciler process namespace create/delete event
 type NamespaceReconciler struct {
-	Client    client.Client
-	Scheme    *apimachineryruntime.Scheme
-	NSXConfig *config.NSXOperatorConfig
+	Client     client.Client
+	Scheme     *apimachineryruntime.Scheme
+	NSXConfig  *config.NSXOperatorConfig
+	VPCService common.IVPCService
```

Comment:
It's good to add comments that why we only use IVPCService here instead of using VPCService

Reply:
It's good rename VPCService as a more meaningful name to help reader understand this purpose of this interface service.